.circleci/config.yml

```diff
@@ -56,6 +60,34 @@ jobs:
           name: Trigger blocks-storybook-app build
           command: echo "TODO Trigger blocks-storybook-app build"
 
+  publish:
+    working_directory: ~/blocks
+    docker:
+      - image: circleci/node:11.10.1
+
+    steps:
+      - checkout
+
+      - run:
+          name: Authenticate npm with registry
+          command: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/repo/.npmrc
```

Comment:
@barbosa @pedrohperalta Do you think it is possible for us to create/use a Loadsmart npm account, so we can automate the release of this library? (Same case of `rn-salesforce-chat`)